README.md

```diff
@@ -42,6 +42,20 @@ Fields that can be included in the response body for `GET /`.
 | description | string          | **REQUIRED.** Detailed multi-line description to fully explain the API. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
 | links       | \[Link Object\] | **REQUIRED.** A list of references to other documents and endpoints. |
 
+##### Relation Types
+
+| Endpoint                               | Relation Type        |
+| -------------------------------------- | -------------------- |
+| `GET /conformance`                     | `conformance`        |
+| `GET /products`                        | `products`           |
+| `GET /products/{productId}`            | `product`            |
+| `GET /products/{productId}/parameters` | `product-parameters` |
+| `GET /orders`                          | `orders`             |
+| `POST /orders`                         | `create-order`       |
+| `GET /orders/{orderId}`                | `order`              |
+| `GET /orders/{orderId}/status`         | `monitor`            |
```

Comment:
Could we use the reltype [status](https://www.rfc-editor.org/rfc/rfc8631.html#page-8)?